go-controller/pkg/ovn/gateway.go

```diff
@@ -390,6 +390,9 @@ func (gw *GatewayManager) GatewayInit(
 			logicalRouterPort.Ipv6RaConfigs = map[string]string{
 				"address_mode": "dhcpv6_stateful",
 			}
+			if gw.netInfo.MTU() > 0 {
+				logicalRouterPort.Ipv6RaConfigs["mtu"] = fmt.Sprintf("%d", gw.netInfo.MTU())
```

Comment:
hmm if user is not providing the value the default value should be provided right? so if MTU is 0 we should use config.Default.MTU?
self-answered:
netconf := &ovncnitypes.NetConf{MTU: Default.MTU}
it seems to be taken care of in the config section already..